megalinter/descriptors/go.megalinter-descriptor.yml

```diff
@@ -19,6 +19,7 @@ linters:
     cli_lint_mode: list_of_files
     cli_lint_extra_args:
       - "run"
+    cli_lint_fix_arg_name: --fix
     cli_version_arg_name: "version"
```

Comment:
Hello,
Minor comment not really related to the PR but I thought I'd mention it since I was working on a monorepo wrapper for golangci lint. I noticed for me the count didn't work with the default unless I added:
```
    cli_lint_errors_count: regex_count
    cli_lint_errors_regex: "\\.go:\\d+:\\d+: .*\\n"
```
That seemed to do it pretty well in my tests.